Apromore-Custom-Plugins/Process-Simulation-Info-Logic/src/main/java/org/apromore/processsimulation/config/SimulationInfoConfig.java

```diff
@@ -36,6 +36,7 @@
 @ConfigurationProperties(prefix = "process-simulation-info-export")
```

Comment:
Can we make SimulationConfig immutable class decorated with ConfigurationProperties?

Reply:
Don't think spring allows that. It requires setters to initialise the configuration at the time of auto wiring

Reply:
I will address the immutability of the properties in a subsequent PR. For now, simplified the properties by removing the usage of maps

Reply:
OK